microsoft/testsuites/core/storage.py

```diff
@@ -112,10 +114,17 @@ def verify_resource_disk_mtab_entry(self, node: RemoteNode) -> None:
             .get_partition_with_mount_point(self.os_disk_mount_point)
             .disk
         )
-        mtab = node.tools[Cat].run("/etc/mtab").stdout
-        resource_disk_from_mtab = get_matched_str(
-            mtab, self._get_mtab_mount_point_regex(resource_disk_mount_point)
-        )
+        if isinstance(node.os, BSD):
+            mtab = node.tools[Mount].run().stdout
+            mtab_mount_point_regex = re.compile(
```

Comment:
This kind of pattern should be in tools. I think we already have some logic to parse mount points, is it right? And the regs should always be with examples.